File: GameCore/src/game/core/IScorable.java
package game.core;

/**
 * Интерфейс для игр в которых можно показать текущий счет.
 *
 * @author <a href="mailto:vladimir.romanov@gmail.com">Romanov V.Y.</a>
 */
public interface IScorable {
    /**
     * Счет для Ыиграющего фигурами заданного цвета.
     *
     * @param color цвет фигур
     * @return счет игрока
     */
    int getScore(PieceColor color);
}

File: GameCore/src/game/core/IScorable.kt
package game.core

/**
 * Интерфейс для игр в которых можно показать текущий счет.
 *
 * @author [Romanov V.Y.](mailto:vladimir.romanov@gmail.com)
 */
interface IScorable {
    /**
     * Счет для играющего фигурами заданного цвета.
     *
     * @param color цвет фигур
     * @return счет игрока
     */
    fun getScore(color: PieceColor): Int
}